src/main/java/com/outbrain/ci/friendly/flatten/maven/plugin/FlattenMojo.java

```diff
@@ -43,15 +45,17 @@
   @Parameter(property = "changelist")
   private String changeList;
 
+  private final PomVisitor pomVisitor = new PomVisitorImpl();
+
   /**
    * {@inheritDoc}
    */
   public void execute() throws MojoExecutionException {
     final String originalPom = readPom();
     final String revision = getRevision();
-    final String modifiedPom = replacePlaceHolders(originalPom, revision, sha1, changeList);
+    final String modifiedPom = pomVisitor.visit(originalPom, revision, sha1, changeList);
     if (originalPom.equals(modifiedPom)) {
-      getLog().info("POM is not CI friendly");
+      getLog().info("POM is not required CI friendly flatten operation");
```

Comment:
Pom does not have any CI friendly properties

Reply:
will change